.github/workflows/static_checks.yml

```diff
@@ -10,6 +10,8 @@ jobs:
 
       - name: Install dependencies
         run: |
+          mkdir -p $HOME/.local/bin
+          export PATH=$HOME/.local/bin:$PATH
```

Comment:
This should likely be an env variable in the outer scope, this won't work in other sub-jobs I think (as your need to call `~/.local/bin/black` manually seems to show).

Reply:
I'm not really sure how to do that, this didn't work:

```yml
env:
  PATH: $HOME/.local/bin:$PATH
```